### PR TITLE
feat: Display Gradle project version

### PR DESCRIPTION
**Pull request by klane**
#### Description

This PR adds package version support for [Gradle](https://gradle.org/) projects. The version number is extracted from the `build.gradle` file in the current directory, similar to existing support for other packages.

#### Motivation and Context

Starship does not currently display Gradle project version numbers.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Screenshots (if appropriate):
starship-gradle-project

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

I added unit tests for `build.gradle` files with and without version numbers. The changes have been tested with bash 5.0.11, zsh 5.7.1, and fish 3.0.2 on macOS High Sierra (10.13.6).

#### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

**Comment by MarioAriasC** [comment]
Good job,

A small suggestion, you can easily add support for `build.gradle.kts` (Kotlin DSL)

Cheers

**Comment by klane** [comment]
@MarioAriasC I have not used `build.gradle.kts` before, but it appears the only change I need to make to the regex is to account for the presence of an equals sign. Does that sound right?

File: src/modules/package.rs [hunk 1]
```diff
@@ -177,4 +188,70 @@ mod tests {
...
+    id 'test.plugin' version '0.2.0'
+}
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}";
+
+        let expected_version = None;
+        assert_eq!(
+            extract_poetry_version(&gradle_without_version),
```
Comment (andytom) [inline src/modules/package.rs]: Should this line be
```suggestion
            extract_gradle_version(&gradle_without_version),
```

**Comment by matchai** [comment]
I got around to addressing the merge conflicts and testing it locally. Works perfectly! 😄 
Thank you @klane and @andytom!

**Comment by klane** [comment]
Thanks @matchai! My apologies for not getting back to this.

**Pull request merged**